### PR TITLE
Pac4j: Fix SAML IdP initiated logout

### PR DESCRIPTION
**Pull request by patari**
Adds simple check that delegation handling may continue correctly for logout requests initiated by identity provider.

Identified the problem when using SAML integration.

### CODE REVIEW

File: support/cas-server-support-pac4j-webflow/src/main/java/org/apereo/cas/web/flow/DelegatedClientAuthenticationAction.java [hunk 1]
```diff
@@ -243,6 +243,10 @@ protected Event handleException(final J2EContext webContext, final BaseClient<Cr
         }
     }
 
+    private boolean isLogoutRequest(final HttpServletRequest request) {
```
Comment (leleuj) [inline support/cas-server-support-pac4j-webflow/src/main/java/org/apereo/cas/web/flow/DelegatedClientAuthenticationAction.java]: This method should be used line 456 as well.

**Comment by patari** [comment]
> I'm in line with the the proposed change. It must be ported forward.

Thanks for the review. I will cherry-pick this into master after approval.

File: support/cas-server-support-pac4j-webflow/src/main/java/org/apereo/cas/web/flow/DelegatedClientAuthenticationAction.java (hunk 1 above)
Comment (patari) [reply support/cas-server-support-pac4j-webflow/src/main/java/org/apereo/cas/web/flow/DelegatedClientAuthenticationAction.java]: Updated line 456.

**Comment by mmoayyed** [comment]
This should be posted as a PR for the master branch before it can be merged.

**Pull request merged**